.gitignore

```diff
@@ -1,11 +1,168 @@
-spacepy.komodoproject
+spacepy.k[ao]modoproject
```

Comment:
There was a typo here, since the package is named "kamodo" rather than "komodo".